src/UKHO.ERPFacade.Common/IO/Azure/AzureTableReaderWriter.cs

```diff
@@ -379,6 +405,35 @@ await foreach (var entity in entities)
             return records.FirstOrDefault();
         }
 
+        public async Task UpdateStatusOfRecordOfSaleEntityPublisedEvent(string correlationId)
```

Comment:
Spell mistakes in method name. please give name as UpdateStatusOfRecordOfSalePublishedEvent

Reply:
Changed the method name as suggested above.